src/App.tsx

```diff
@@ -10,32 +11,27 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState('tab-1');
+  const isSelectedTab = tabs.find((tab) => selectedTabId === tab.id)
+    || tabs[0];
```

Comment:
why not just store `selectedTab` here, you're passing whole object to the handler anyway. This way you won't need to do `tabs.find` in both components